sherpa/astro/data.py

```diff
@@ -454,6 +467,16 @@ def get_ylabel(self):
         return 'Counts'
 
 
+# FIXME There are places in the code that explicitly check if an object is an instance of sherpa.astro.data.DataRMF.
+# So it's safer to make DataRosatRMF a subclass of the default class, although in principle they should be siblings
+# and subclasses of the same superclass.
+class DataRosatRMF(DataRMF):
```

Comment:
In principle this is `DataNoValidationRMF`, which is subclassed to `DataRosatRMF`, but I think that's over-engineering for now (the thought being that there could be other cases where we decide to do no validation).

Reply:
In principle, I agree.